app/src/main/java/a/a/a/yq.java

```diff
@@ -337,66 +337,66 @@ public void a(String iconPath) {
      */
     public void a(Context context) {
         boolean logcatEnabled = N.isDebugBuild && new BuildSettings(sc_id).getValue(
-                BuildSettings.SETTING_ENABLE_LOGCAT, BuildSettings.SETTING_GENERIC_VALUE_TRUE).equals(BuildSettings.SETTING_GENERIC_VALUE_TRUE);
+            BuildSettings.SETTING_ENABLE_LOGCAT, BuildSettings.SETTING_GENERIC_VALUE_TRUE).equals(BuildSettings.SETTING_GENERIC_VALUE_TRUE);
 
         String javaDir = FileUtil.getExternalStorageDir() + "/.sketchware/data/" + sc_id + "/files/java/";
         if (!new File(javaDir, "DebugActivity.java").exists()) {
             fileUtil.b(javaFilesPath + File.separator
-                            + packageNameAsFolders + File.separator
-                            + "DebugActivity.java",
-                    PACKAGE_PLACEHOLDER_PATTERN.matcher(fileUtil.b(
-                            context,
-                            "debug" + File.separator
-                                    + "DebugActivity.java"
-                    )).replaceAll(packageName));
+                       + packageNameAsFolders + File.separator
+                       + "DebugActivity.java",
+                       PACKAGE_PLACEHOLDER_PATTERN.matcher(fileUtil.b(
+                                                               context,
+                                                               "debug" + File.separator
+                                                               + "DebugActivity.java"
+                                                           )).replaceAll(packageName));
         }
 
         String customApplicationClassName = new ProjectSettings(sc_id).getValue(ProjectSettings.SETTING_APPLICATION_CLASS,
-                ".SketchApplication");
+                                                                                ".SketchApplication");
         boolean notUsingCustomApplicationClass = customApplicationClassName.equals(".SketchApplication");
         if (!new File(javaDir, "SketchApplication.java").exists() && notUsingCustomApplicationClass) {
             boolean applyMultiDex = projectSettings.getMinSdkVersion() < 21;
 
             String sketchApplicationFileContent = PACKAGE_PLACEHOLDER_PATTERN.matcher(fileUtil.b(
-                    context,
-                    "debug" + File.separator + "SketchApplication.java"
-            )).replaceAll(packageName);
+                                                                                          context,
+                                                                                          "debug" + File.separator + "SketchApplication.java"
+                                                                                      )).replaceAll(packageName);
```

Comment:
note that whitespace changes like this automatically get undone by reformatting the file